docs/modules/ROOT/nav.adoc

```diff
@@ -1 +1,23 @@
-* xref:index.adoc[Quarkus Renarde]
+include::./includes/attributes.adoc[]
+:config-file: application.properties
+
+* xref:index.adoc[Getting started]
```

Comment:
Can't we generate this TOC? It's bound to be forgotten/out of sync, no?

Reply:
@gastaldi ? Could generate it from the `==` of all the pages?

Reply:
Maybe there is an Asciidoc macro or something that will generate the TOC based on the file. I know there is this: https://docs.asciidoctor.org/asciidoc/latest/toc/position/#at-macro but it doesn't seem to work for XREF links